### ISSUE TITLE
Switch to rasterio 1.0 on Travis and cluster?

### ISSUE THREAD

**Issue by fmaussion** [OGGM/oggm#473]
the last stable release of rasterio was 0.36 on June 15, 2016. rasterio 1.0b1 was released 20 days ago.

We have always tried to support both versions but now there is something I need which is only available in rasterio 1+ (https://github.com/mapbox/rasterio/issues/1295 fails in https://travis-ci.org/OGGM/oggm/jobs/391369396).

I'm ok to leave 0.36 on Travis in order to keep the pace with users who might have the older version but I an super keen in having rasterio 1+ on cluster. What do you think @TimoRoth ?

**Comment by TimoRoth** [comment]
It's not something that's globally installed, but it's in every users venc.
So the two can happy co-exist depending on user needs.
pip just won't install beta versions by default.

I can switch the travis container over if you like, that's easily done.

**Comment by fmaussion** [comment]
> I can switch the travis container over if you like, that's easily done. 

No, I think its better to keep supporting the two versions for a while. I'll install V1.0 on cluster though.

**Comment by fmaussion** [comment]
Yes, so rasterio 1.0b already caused a lot of warnings and the necessity to define a new global variable.

Let's be honest: this rasterio 0.36 -> 1.0 is very long and painful, I hope that they are going to finish this soon...
